docs/.eleventy.js

```diff
@@ -49,6 +52,7 @@ module.exports = function(eleventyConfig) {
 
     eleventyConfig.addGlobalData("site_name", siteName);
     eleventyConfig.addGlobalData("GIT_BRANCH", process.env.BRANCH);
+    eleventyConfig.addGlobalData("HEAD", process.env.BRANCH !== "latest" || process.env.BRANCH !== "next");
```

Comment:
```suggestion
    eleventyConfig.addGlobalData("HEAD", process.env.BRANCH !== "latest" && process.env.BRANCH !== "next");
```

With `||`, it would always be `true`?